go/signedexchange/signedexchange.go

```diff
@@ -51,12 +51,69 @@ func NewExchange(uri *url.URL, requestHeaders http.Header, status int, responseH
 	}, nil
 }
 
+// RequestURI returns the exchange's request URI.
+// The caller should not modify the returned content.
+func (e *Exchange) RequestURI() *url.URL {
+	return e.requestUri
+}
+
+// RequestHeaders returns the exchange's request headers.
+// The caller should not modify the returned content.
+func (e *Exchange) RequestHeaders() http.Header {
+	return e.requestHeaders
+}
+
 // Payload returns the slice holding the payload.
 // The caller should not modify the returned content.
 func (e *Exchange) Payload() []byte {
 	return e.payload
 }
 
+type DumpOptions struct {
+	DumpContentText bool
+}
+
+func (e *Exchange) Dump(w io.Writer, opts DumpOptions) error {
```

Comment:
Why do we need to introduce a struct for a option? Isn't just an argument enough so far? My expectation is `Dump`'s signature is changeable though `Dump` is exported.